test/release/examples/primers/modules.chpl

```diff
@@ -6,7 +6,7 @@
 
     * how to define a module
     * namespace control within a module
-    * external access to a module's symbols
+    * access to a module's symbols
```

Comment:
I would append "from outside the module" to preserve the original meaning (since I'm assuming you felt external was potentially confusing with `extern` or something?)